docs/UserReward.rst

```diff
@@ -0,0 +1,132 @@
+.. _UserReward:
+
+#####################
+Managing your reward
+#####################
+
+=======
+Wallet
+=======
+
+
+General information
+~~~~~~~~~~~~~~~~~~~~
+
+Cryto-monnaie
+A cryptocurrency (or crypto currency) is a digital asset designed to work as a medium of exchange that uses strong cryptography to secure financial transactions, control the creation of additional units, and verify the transfer of assets.
+
+A cryptocurrency wallet is a software program that stores private and public keys and interacts with various blockchain to enable users to send and receive digital currency and monitor their balance. 
+If you want to use Bitcoin or any other cryptocurrency, you will need to have a digital wallet.
+
+
+
+How to access to Wallet interface
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+(old : To access to wallet's interface, you have to go to:)
+
+Before creating your wallet you have to access to dedicated interface. There is three ways to display wallet's interface :
+
+
+1- From your display name on the top navigation bar then "My wallet" from the drop-down menu.
+
+|image0|
+
+2- From your profile then "My wallet" application.
+
+|image1|
+
+
+3- From "My application" of the left navigation then "My wallet".
+
+|image2|
+
+
+How to create your wallet
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first thing to do is to create your wallet by setting a password. The password must contain at least 8 caracters. 
+
+|image3|
+
+Once your password is set, your wallet is created but you will not be allowed to make transactions until an administrator approves it. The wallet's interface is nevertheless available and you may request funds from other users.
```

Comment:
```suggestion
Once your password is set, your wallet is created but you will not be allowed to use it until an administrator approves it. The wallet's interface is nevertheless available and you may request funds from other users.
```

Reply:
I don't think we should allow this because incoming funds transactions will fail until the wallet is approved. 
@boubaker right ?